test/ssl-tests/04-client_auth.conf.in

```diff
@@ -26,15 +26,23 @@ sub generate_tests() {
     foreach (0..$#protocols) {
         my $protocol = $protocols[$_];
         my $protocol_name = $protocol || "flex";
+        my $caalert;
         if (!$is_disabled[$_]) {
+            if ($protocol_name eq "SSLv3") {
```

Comment:
Hypothetically, "flex" would end up negotiating SSLv3 if that is the only enabled protocol.

However, I'm not entirely sure we should bother testing this contrived configuration. Up to you how correct you want to be there. Possibly more things break in that config anyway.